sympy/physics/vector/frame.py

```diff
@@ -420,72 +421,211 @@ def dcm(self, otherframe):
-        successive simple rotations. Acceptable rotation orders are of length
-        3, expressed in XYZ or 123, and cannot have a rotation about about an
-        axis twice in a row.
+        Axis
+        ----
 
-        >>> B.orient(N, 'Body', [q1, q2, q3], 123)
-        >>> B.orient(N, 'Body', [q1, q2, 0], 'ZXZ')
-        >>> B.orient(N, 'Body', [0, 0, 0], 'XYX')
+        ``rot_type='Axis'`` creates a direction cosine matrix defined by a
+        simple rotation about a single axis fixed in both reference frames.
+        This is a rotation about an arbitrary, non-time-varying
+        axis by some angle. The axis is supplied as a Vector. This is how
+        simple rotations are defined.
 
-        Next is Space. Space is like Body, but the rotations are applied in the
-        opposite order.
+        >>> B.orient(N, 'Axis', (q1, N.x))
 
-        >>> B.orient(N, 'Space', [q1, q2, q3], '312')
+        The ``orient()`` method generates a direction cosine matrix and its
+        transpose which defines the orientation of B relative to N and vice
+        versa. Once orient is called, ``dcm()`` outputs the appropriate
+        direction cosine matrix.
+        >>> B.dcm(N)
+        Matrix([
+        [1,       0,      0],
+        [0,  cos(q1), sin(q1)],
+        [0, -sin(q1), cos(q1)]])
 
-        Next is Quaternion. This orients the new ReferenceFrame with
-        Quaternions, defined as a finite rotation about lambda, a unit vector,
-        by some amount theta.
-        This orientation is described by four parameters:
-        q0 = cos(theta/2)
-        q1 = lambda_x sin(theta/2)
-        q2 = lambda_y sin(theta/2)
-        q3 = lambda_z sin(theta/2)
-        Quaternion does not take in a rotation order.
+        The following two lines show how the sense of the rotation can be
+        defined. Both lines produce the same result.
+        >>> B.orient(N, 'Axis', (q1, -N.x))
+        >>> B.orient(N, 'Axis', (-q1, N.x))
 
-        >>> B.orient(N, 'Quaternion', [q0, q1, q2, q3])
+        The axis does not have to be defined by a unit vector, it can be any
+        vector in the parent frame.
 
-        Next is Axis. This is a rotation about an arbitrary, non-time-varying
-        axis by some angle. The axis is supplied as a Vector. This is how
-        simple rotations are defined.
+        >>> B.orient(N, 'Axis', (q1, N.x + 2 * N.y))
+
+        DCM
+        ---
+
+        The direction cosine matrix can be set directly. The orientation of a
+        frame A can be set to be the same as the frame B above like so:
+
+        >>> B.orient(N, 'Axis', (q1, N.x))
+        >>> A = ReferenceFrame('A')
+        >>> A.orient(N, 'DCM', N.dcm(B))
+        >>> A.dcm(N)
+        Matrix([
+        [1,       0,      0],
+        [0,  cos(q1), sin(q1)],
+        [0, -sin(q1), cos(q1)]])
+
+        **Note carefully that ``N.dcm(B)`` was passed into ``orient()`` for
+        ``A.dcm(N)`` to match ``B.dcm(N)``.**
```

Comment:
I don't think that emphasis can be used with inline literals.

![image](https://user-images.githubusercontent.com/34944973/67186495-29925780-f423-11e9-8de5-304d831cf506.png)

Reply:
Fixed, thanks.